lib/game.rb

```diff
@@ -41,14 +41,22 @@ def place_user_ships
   end
 
   def play_game
-    # we need to create a loop to run through display_boards,
-    # computer_player_turn and user_player_turn until someone loses
-
-  end
-
-  def game_over
-    # @turn.user_player_ship_count == 2
-    # need to write logic for this, ship_count goes up when ship is sunk
-    # for either user
+    loop do
+      @turn.display_boards
+      @turn.user_player_turn(user_player)
+      @turn.computer_player_turn(computer_player)
+
+      if @turn.computer_player_ship_count == 2
+        puts @turn.menu.computer_won
+        self.start
+        break
+      elsif @turn.user_player_ship_count == 2
+        puts @turn.menu.user_won
+        self.start
+        break
+      else
+        next
+      end
+    end
```

Comment:
I love the simplicity of this loop! Really like we use the ship_counts to determine who won and go back to the start method.